addons/l10n_de/models/account_move.py

```diff
@@ -22,5 +19,5 @@ def _compute_show_delivery_date(self):
     def _post(self, soft=True):
         for move in self:
```

Comment:
```suggestion
    def _post(self, soft=True):
        posted = super()._post(soft)
        for move in self:
```
If we call `super()` first, we can rely on the fact that the `invoice_date` will be set, so we don't have to provide a fallback value.
I personally would prefer that, but it's mostly subjective, so up to you 😊